source_code/gfm-v2.js

```diff
@@ -1389,7 +1389,7 @@ function displayFloodImpactPortal(aoi) {
   
   // After calculating the affected population, remove zero values from the population layers
   function mask_pop(image){
-    return image.updateMask(image.gte(0)).selfMask();
+    return image.updateMask(image).selfMask();
```

Comment:
@AarnavAgrawal what is this line doing? The initial code was masking out all the zero value cells. But it we remove `.gte(0)` then masking an image with its own mask won't cause anything.